modules/appeals_api/app/workers/appeals_api/pdf_submit_job.rb

```diff
@@ -67,7 +74,7 @@ def upload_to_central_mail(appeal, pdf_path)
     end
 
     def process_response(response, appeal, metadata)
-      if response.success? || response.body.match?(NON_FAILING_ERROR_REGEX)
+      if response.success?
```

Comment:
👍

Reply:
Haha yea, that line fixes the original issue of the DUP Guid attempted SC submission of not being marked as error in the sc tracking table.
I think this is the only change absolutely needed to fix the error....the rest of these changes are related to re-running upload attempts that hit what might be a transitory issue...